bin/src/app.rs

```diff
@@ -232,7 +232,9 @@ fn config_dir(app: &tauri::App, args: &Args) -> PathBuf {
 }
 
 async fn should_start_main_window(args: &Args) -> bool {
-    if args.hidden {
+    if let Some(Commands::App { hidden }) = &args.command
+        && *hidden
+    {
```

Comment:
here, I think we should just receive the `Commands::App` element, because it should be always true that the command itself is that one

so the argument of this function should change, not this condition